InnerEye/ML/configs/segmentation/Hippocampus.py

```diff
@@ -0,0 +1,68 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from typing import Any
+
+import pandas as pd
+
+from InnerEye.ML.config import ModelArchitectureConfig, PhotometricNormalizationMethod, SegmentationModelBase, \
+    equally_weighted_classes, SegmentationLoss
+from InnerEye.ML.deep_learning_config import OptimizerType
+from InnerEye.ML.utils.split_dataset import DatasetSplits
+
+MODALITY = "mri"
+LABELS = ["hippocampus_L", "hippocampus_R"]
+
+class Hippocampus(SegmentationModelBase):
+    """
+    Hippocampus segmentation in MR scans.
+    """
```

Comment:
These three lines should be merged: `"""Hippocampus segmentation in MR scans."""`

Reply:
Done, thanks!